tests/testStreamOpen.cpp

```diff
@@ -331,6 +331,17 @@ TEST_F(StreamOpenOutput, PlaybackFormatFloatReturnsErrorBeforeLollipop){
     }
 }
 
+TEST_F(StreamOpenOutput, PlaybackFormatFloatReturnsFloatBeforeLollipopWithFormatConversionAllowed){
+    if (getSdkVersion() < __ANDROID_API_L__){
```

Comment:
We might well allow this test to run on every version. It should pass on every version.

Reply:
That's a broader scoped test though, since on L+ a float stream should be returned regardless of whether format conversion is allowed. This test is specifically for pre-L.

Reply:
I don't understand why the conditional is needed. The test should pass on every version of Android including pre-L. But OK.

Reply:
Yeah, actually you're right. I was thinking about it simply from the point of view that pre-L devices *must* have format conversion enabled to return a float stream, whereas L+ devices it shouldn't matter. There could be a scenario where future Android versions default to a different format (`I32`?) in which case this test should cover that code path. I will address in a follow up CL (reminder: #1270).